pavelib/assets.py

```diff
@@ -165,7 +165,7 @@ def collect_assets(systems, settings):
     `settings` is the Django settings module to use.
     """
     for sys in systems:
-        sh(django_cmd(sys, settings, "collectstatic --noinput > /dev/null"))
+        sh(django_cmd(sys, settings, "collectstatic --clear --noinput > /dev/null"))
```

Comment:
It looks like `REQUIRE_DEBUG` doesn't affect compression time (https://github.com/etianen/django-require/search?utf8=%E2%9C%93&q=REQUIRE_DEBUG).  Maybe we should use `pipeline.storage.PipelineCachedStorage` on devstack and `django_require.staticstorage.OptimizedCachedRequireJsStorage` on production?